qa/L0_batch_input/batch_input_test.py

```diff
@@ -154,27 +173,31 @@ def test_element_count(self):
                         idx, expected_value, output_data))
         except InferenceServerException as ex:
             self.assertTrue(False, "unexpected error {}".format(ex))
+        self.client.stop_stream()
 
     def test_accumulated_element_count(self):
         model_name = "ragged_acc_shape"
+        user_data = queue.Queue()
+        self.client.start_stream(
+            callback=partial(self.client_callback, user_data))
 
         output_name = 'BATCH_AND_SIZE_OUTPUT'
-        outputs = [tritonhttpclient.InferRequestedOutput(output_name)]
+        outputs = [grpcclient.InferRequestedOutput(output_name)]
 
         async_requests = []
         try:
             for inputs in self.inputs:
```

Comment:
same here: could you please clarify why self.inputs here was not replaced as in other places?